experimental/packages/opentelemetry-sdk-metrics/src/export/AggregationSelector.ts

```diff
@@ -29,4 +29,17 @@ export type AggregationSelector = (instrumentType: InstrumentType) => Aggregatio
 export type AggregationTemporalitySelector = (instrumentType: InstrumentType) => AggregationTemporality;
 
 export const DEFAULT_AGGREGATION_SELECTOR: AggregationSelector = _instrumentType => Aggregation.Default();
-export const DEFAULT_AGGREGATION_TEMPORALITY_SELECTOR: AggregationTemporalitySelector = _instrumentType => AggregationTemporality.CUMULATIVE;
+export const CUMULATIVE_AGGREGATION_TEMPORALITY_SELECTOR: AggregationTemporalitySelector = _instrumentType => AggregationTemporality.CUMULATIVE;
+export const DELTA_AGGREGATION_TEMPORALITY_SELECTOR: AggregationTemporalitySelector = (instrumentType: InstrumentType) => {
```

Comment:
I didn't realize that spec was specific to the OTLP exporter but I think that's only because it is the only specified exporter which supports multiple temporalities. I think we can clarify this in the spec though and it doesn't block the GA release. Adding delta support later shouldn't be a breaking change.

Reply:
Oops, you're right, I think it is not defined in the SDK spec. I can change this to take a selector instead when I'm back in the office tomorrow.